tests/conftest.py

```diff
@@ -2505,15 +2505,20 @@ def cloud_data_context_with_datasource_pandas_engine(
                 - default_identifier_name
         """,
     )
-    context.add_datasource(
-        "my_datasource",
-        **config,
-    )
+    with mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend.list_keys"
+    ), mock.patch(
+        "great_expectations.data_context.store.ge_cloud_store_backend.GeCloudStoreBackend._set"
+    ):
```

Comment:
Any value in `with mock.patch(...) as mock_xyz` and then asserting that `mock_xyz` has been called or used?

Reply:
Not really because the behavior of the datasource store should be tested here instead of fixtures: https://github.com/great-expectations/great_expectations/blob/develop/tests/data_context/cloud_data_context/test_datasource_crud.py